### ISSUE TITLE
Check that viewport zoom is not disabled

### ISSUE THREAD

**Issue by techgique** [CDRH/orchid#123]
https://a11yproject.com/checklist/#section-global-code

I think such disabling might be commonly done in framework templates and recommended for the "native app-like experience" mentioned in the note in a11yproject's checklist. Want to make sure we handle this for a11y

**Comment by jduss4** [comment]
It's possible I'm missing something, like if you noticed this was amiss, but I think this should be doing what we need it to do already?

`<meta name="viewport" content="width=device-width, initial-scale=1">`

I checked MDN (https://developer.mozilla.org/en-US/docs/Web/CSS/Viewport_concepts) and it says:

> The width property controls the size of the viewport. It should preferably be set to device-width, which is the width of the screen in CSS pixels at a scale of 100%. There are other properties, including maximum-scale, minimum-scale, and user-scalable, which control whether users can zoom the page in or out, but the default values are the best for accessibility and user experience, so these can be omitted.

**Comment by techgique** [comment]
I just wanted to double-check. The code looks good and zooming works on mobile for me :+1:
